app/controllers/idv/welcome_controller.rb

```diff
@@ -52,6 +53,13 @@ def analytics_arguments
       }.merge(ab_test_analytics_buckets)
     end
 
+    def delete_old_capture_session_if_present
```

Comment:
do we need to delete the session? ... a more minimalist approach could be to set the DocumentCaptureSession.result_id to nil  in hybrid handoff when redo params is true... Ultimately, I'm curious how are handling this in our other documentcapture controllers.